README.rst

```diff
@@ -0,0 +1,303 @@
++======================+==============+=================+=================+=================+=================+
+| BDW (Broadwell)      | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| SKL (Skylake)        | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| BXT (Broxton)        | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| APL (Apollo Lake)    | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| KBLx [1]             | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ICL (Ice Lake)       | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| JSL (Jasper Lake)    | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| EHL (Elkhart Lake)   | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| TGL (Tiger Lake)     | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| DG1 (Xe MAX)         | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| SG1                  | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| RKL (Rocket Lake)    | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ADL-S (Alder Lake S) | Legacy       | ✔               | ✔               | ✔               |                 |
```

Comment:
We need to remove ADL support on MediaSDK because it will be covered by VPL

Reply:
@aobolensk , please fix

Reply:
Done. Removed support for ADL-S and fixed README